metafix-vsc/fix.snippets.json

```
@@ -1,7 +1,22 @@
 {
-    "do bind": {
-      "prefix": ["do"],
-      "body": ["do ${1:collector}(${2:args})", "\t${0:body}", "end"],
-      "description": "A do bind changes the execution context of the body functions. They are executed for each element in the list."
+    "do list": {
+      "prefix": ["do list"],
+      "body": ["do list(path: \"${1:<sourceField>}\", \"var\": \"${4:<variableName>}\")", "\t${0:body}", "end"],
+      "description": "Iterates over each element of an array. In contrast to Catmandu, it can also iterate over a single object or string."
+    },
+    "do list_as": {
+      "prefix": ["do list_as"],
+      "body": ["do list_as(\"${1:<variableName_1>}\", \"${2:<sourceField_1>}\")", "\t${0:body}", "end"],
```

```suggestion
      "body": ["do list_as(\"${1:<variableName_1>}\": \"${2:<sourceField_1>}\")", "\t${0:body}", "end"],
```

As in:

```perl
do list_as("variableName_1": "sourceField_1")
```

(See [documentation](https://github.com/metafacture/metafacture-fix?tab=readme-ov-file#do-list_as).)